packages/design-system/src/components/utilities/typographyBodyText.stories.tsx

```diff
@@ -27,6 +27,24 @@ export const AllBodyText = () => {
     </>
   );
 };
+export const AllBodyTextOnDark = () => {
+  return (
+    <>
+      <p className="ds-text-body--lg">
+        <strong>Large:</strong> {body}
+      </p>
+      <p className="ds-text-body--md">
+        <strong>Medium:</strong> {body}
+      </p>
+      <p className="ds-text-body--sm">
+        <strong>Small:</strong> {body}
+      </p>
+    </>
+  );
+};
```

Comment:
Same thing here where the body of the story doesn't change, so we can actually just make a copy of the story this way:

```suggestion
export const AllBodyTextOnDark = AllBodyText.bind({});
```

The `.bind({})` is a bit ugly, but it's what they do in Storybook docs, I'm pretty sure. It's JavaScript for _clone this function, please_.